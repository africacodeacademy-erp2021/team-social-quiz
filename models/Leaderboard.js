const mongoose = require("mongoose");
const ObjectId = mongoose.Schema.Types.ObjectId;

const LeaderboardSchema = new mongoose.Schema(
  {

    user:{
      type:ObjectId,
      ref: "User",
    },
    
    score:{
        type: ObjectId,
        ref: "Game",
    },

  },
  {
    
    timestamps: true,
    
  }
);

const Leaderboard = mongoose.model("Leaderboard", LeaderboardSchema);

module.exports = Leaderboard;
