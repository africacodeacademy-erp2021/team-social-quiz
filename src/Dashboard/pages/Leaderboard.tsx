import React from 'react'
import Navbar from '../components/Navbar';

function Leaderboard() {
    return (
        <div className="screen">
        <div className="hamburger">
          <Navbar />
        </div>
        <div className="sidebar">
            <h1>Leaderboard</h1>
            </div>
        </div>
      
    )
}
export default Leaderboard;