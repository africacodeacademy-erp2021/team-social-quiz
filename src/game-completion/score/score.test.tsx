//import {screen } from '@testing-library/react';
//import { render } from 'react-dom';
//import ScoreBoard from './score';
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { configure, shallow } from "enzyme";
import Leaderboard from '../leaderboard/leaderboard';
import ScoreBoard from "./score";

 
configure({ adapter: new Adapter() });

describe("leaderboard component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Leaderboard />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("ScoreBored compornent", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<ScoreBoard />);
    expect(wrapper).toMatchSnapshot();
  });
});
 
  



