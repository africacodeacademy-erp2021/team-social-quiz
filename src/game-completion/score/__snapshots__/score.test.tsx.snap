// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` renders correctly 1`] = `ShallowWrapper {}`;

exports[`Score component renders correctly 1`] = `ShallowWrapper {}`;

exports[`Score component renders correctly 2`] = `ShallowWrapper {}`;

exports[`Score renders correctly 1`] = `ShallowWrapper {}`;

exports[`ScoreBored   renders correctly 1`] = `ShallowWrapper {}`;

exports[`ScoreBored c renders correctly 1`] = `ShallowWrapper {}`;

exports[`ScoreBored comporn  renders correctly 1`] = `ShallowWrapper {}`;

exports[`ScoreBored compornent  renders correctly 1`] = `ShallowWrapper {}`;

exports[`ScoreBored compornent renders correctly 1`] = `ShallowWrapper {}`;

exports[`c renders correctly 1`] = `ShallowWrapper {}`;

exports[`le component renders correctly 1`] = `ShallowWrapper {}`;

exports[`leaderboa component renders correctly 1`] = `ShallowWrapper {}`;
